test/e2e/kibishii_tests.go

```diff
@@ -144,6 +145,13 @@ func runKibishiiTests(client testClient, providerName, veleroCLI, veleroNamespac
 		return errors.Wrapf(err, "failed to delete namespace %s", kibishiiNamespace)
 	}
 
+	// the snapshots of AWS may be still in pending status when do the restore, wait for a while
+	// to avoid this https://github.com/vmware-tanzu/velero/issues/1799
+	// TODO remove this after https://github.com/vmware-tanzu/velero/issues/3533 is fixed
+	if providerName == "aws" && useVolumeSnapshots {
```

Comment:
Would the other providers benefit from waiting here as well? I'm not sure how long vSphere/Azure snapshots take. Are you only seeing issues with AWS?

Reply:
@zubron I know from prior experience with the snapshot wait issue that it's definitely a concern for Azure and GCP as well. When we implemented a workaround for this in our konveyor forks for the plugins, the fix was needed for all 3.

Reply:
Thanks for the info, @sseago. I'll open another issue to follow up on this and wait for all snapshots.